scalding-core/src/test/scala/com/twitter/scalding/StringUtilityTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.twitter.scalding
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class StringUtilityTest extends WordSpec with Matchers {
```

Comment:
yes, that's what I meant.

assert that it matches `String.split(Arbitrary[String], OneOf(",", "foo", ":"), -1)`

the -1 makes split behave the way you'd expect, eg, just splits on the delimiter, includes empty strings when there are double delimiters and so on.